addons/events/fnc_playerEvent.sqf

```diff
@@ -113,7 +113,7 @@ if !(_state isEqualTo GVAR(oldState)) then {
         } forEach [primaryWeaponMagazine _unit, secondaryWeaponMagazine _unit, handgunMagazine _unit];
 
         if !(_newLoadoutNoAmmo isEqualTo GVAR(oldLoadoutNoAmmo)) then {
-            [QGVAR(loadoutEvent), [_unit, GVAR(oldLoadout), _newLoadout]] call CBA_fnc_localEvent;
+            [QGVAR(loadoutEvent), [_unit, _newLoadout, GVAR(oldLoadout)]] call CBA_fnc_localEvent;
```

Comment:
But loadout was old,new before (it only returned the old array before).

Reply:
https://github.com/CBATeam/CBA_A3/blob/43ac212eafe89ac8cd1fe7e49cf2c2f2548c6690/addons/events/fnc_addPlayerEventHandler.sqf
the var was "old", but it was updated before the event was triggered
```
GVAR(oldLoadout) = _data;
...
[QGVAR(loadoutEvent), [_player, GVAR(oldLoadout)]] call CBA_fnc_localEvent;
```

Reply:
Then I need to change the wiki.

Reply:
Okay, then I need to update the wiki.

Reply:
Wiki got changed.